### DISCUSSION TITLE
Alfa TV [EC]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#379] [Show and tell]
### Channel ID

AlfaTV.ec

### Stream URL

https://byecableiptv.com/alfatv/alfatv.stream/playlist.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Stream does not work (geo-blocked)?

**Comment by freearhey** [comment]
https://byecableiptv.com/alfatv/alfatv.stream/playlist.m3u8 (offline)
<img width="406" alt="image" src="https://user-images.githubusercontent.com/7253922/232255221-0cb5f643-dd64-4453-b53c-624be4a8bfe7.png">
